services/autoscaling/src/simcore_service_autoscaling/modules/auto_scaling_core.py

```diff
@@ -470,14 +470,14 @@ async def _start_warm_buffer_instances(
     with log_context(
         _logger, logging.INFO, f"start {len(instances_to_start)} buffer machines"
     ):
+        started_instances = await get_ec2_client(app).start_instances(
+            instances_to_start
+        )
+        # NOTE: first start the instance and then set the tags in case the instance cannot start (e.g. InsufficientInstanceCapacity)
         await get_ec2_client(app).set_instances_tags(
             instances_to_start,
```

Comment:
shound you use `started_instance` here instead of `instances_to_start`? As far as I underatnd `started_instances != instances_to_start`

Reply:
correct. since I inverted them I missed it. thanks!